.buildkite/heartbeat/heartbeat-pipeline.yml

```diff
@@ -110,11 +121,13 @@ steps:
   - group: "Heartbeat Extended Testing MacOS"
     key: "heartbeat-extended-tests-macos"
     if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*macOS.*/
+
     steps:
       - label: ":mac: Heartbeat MacOS Unit Tests"
         key: "macos-extended"
-        if: build.env("GITHUB_PR_TRIGGER_COMMENT") == "heartbeat for macos" || build.env("GITHUB_PR_LABELS") =~ /.*macOS.*/
-        command: "cd ${BEATS_PROJECT_NAME} && mage build unitTest"
+        command: |
+          cd heartbeat
+          mage build unitTest
         agents:
           provider: "orka"
           imagePrefix: "${IMAGE_MACOS_X86_64}"
```

Comment:
can we also add macos arm64 too, since it's supported in the corresponding [Jenkinsfile](https://github.com/elastic/beats/blob/main/.buildkite/hooks/pre-command#L14)? see https://github.com/elastic/beats/blob/main/.buildkite/metricbeat/pipeline.yml#L234-L249 for an example